numerics/poisson_series_basis_body.hpp

```diff
@@ -230,12 +230,11 @@ std::array<PoissonSeriesSubspace, sizeof...(indices)> PeriodicSeriesGenerator<
       PoissonSeriesSubspace{
           static_cast<PoissonSeriesSubspace::Coordinate>(indices % dimension),
           // The parity of the trigonometric factors of the ith basis element is
-          // (i / dimension) % 2; the degrees its polynomial factor is
-          // i / (2 * dimension), so that the overall parity of that basis
-          // element is (i / dimension + i / (2 * dimension)) % 2, which
-          // simplifies to (3 * i / (2 * dimension)) % 2.
+          // ⌊i / dimension⌋ mod 2; the degrees its polynomial factor is
```

Comment:
The degree, singular (this is remnant from an unsuccessful attempt to phrase this in terms of `indices`).